source/features/one-click-review-submission.tsx

```diff
@@ -32,7 +31,9 @@ function replaceCheckboxes(originalSubmitButton: HTMLButtonElement): void {
 
 	// Generate the new buttons
 	for (const radio of radios) {
-		const tooltip = radio.parentElement!.getAttribute('aria-label');
+		const labelNode = radio.labels?.[0];
+
+		const tooltip = labelNode?.nextElementSibling?.textContent;
```

Comment:
I've also just amended this so that we get those caption descriptions in the button tooltips.

```html
<div class="FormControl-radio-wrap">
  <input aria-describedby="validation-a44ca7e3-1a4e-40ba-9969-cac56e1cf44b caption-a44ca7e3-1a4e-40ba-9969-cac56e1cf44b" class="FormControl-radio" type="radio" value="comment" checked="checked" name="pull_request_review[event]" id="pull_request_review[event]_comment">
  <span class="FormControl-radio-labelWrap">
    <label class="FormControl-label" for="pull_request_review[event]_comment">
      Comment
    </label>       
     <span class="FormControl-caption" id="caption-a44ca7e3-1a4e-40ba-9969-cac56e1cf44b">Submit general feedback without explicit approval.</span>

  </span>
</div>
```

Again, it would possibly be less brittle to select another way, like using the describedby IDs to get the elements that truly describe them, but that requires us to build another query. WDYT?

Reply:
![image](https://github.com/refined-github/refined-github/assets/609062/f08aecce-fb29-4ae9-a661-37a91e58472b)
![image](https://github.com/refined-github/refined-github/assets/609062/5413beee-e258-4a0a-b7ec-7242643699a3)
![image](https://github.com/refined-github/refined-github/assets/609062/ed3b33bc-c8c8-4291-abed-6e3bcb337b7d)

Reply:
Why did the previous version not work?

Reply:
Nothing - the radio's parent (see the `div` above) doesn't have an `aria-label` attribute anymore because it's now properly labelled by the linked `<label>` element